pkg/agent/util/net_windows.go

```diff
@@ -106,6 +107,14 @@ func RemoveManagementInterface(networkName string) error {
 	return err
 }
 
+// ConfigureMacAddress set specified MAC address on interface
+func ConfigureMacAddress(ifaceName string, macConfig net.HardwareAddr) error {
```

Comment:
ConfigureMACAddress()
MAC should be all upper case.

Reply:
Done